projects/plugins/boost/app/assets/src/js/api/speed-scores.ts

```diff
@@ -157,48 +157,66 @@ export function getScoreLetter( mobile: number, desktop: number ): string {
 }
 
 /**
- * Find out if scores were improved.
- *
- * Only show the speed scores if there was an improvement on either mobile or desktop, and neither worsened.
+ * Find out if site scores changed. We fire a popout modal if they improve or worsen.
+ * The message varies depending on the results of the speed scores so lets modify this
  *
  * @param {SpeedScoresSet} scores
  * @return boolean
  */
-export function didScoresImprove( scores: SpeedScoresSet ): boolean {
+export function didScoresChange( scores: SpeedScoresSet ): boolean {
 	const current = scores.current;
 	const noBoost = scores.noBoost;
 
-	// Consider the score was improved if either desktop or mobile improved and neither worsened.
 	return (
 		null !== current &&
 		null !== noBoost &&
-		current.mobile >= noBoost.mobile &&
-		current.desktop >= noBoost.desktop &&
-		current.mobile + current.desktop > noBoost.mobile + noBoost.desktop &&
-		( getScoreImprovementPercentage( scores ) >= 5 || current.desktop + current.mobile > 180 )
+		current.mobile !== noBoost.mobile &&
+		current.desktop !== noBoost.desktop
```

Comment:
Do we want an or operation here instead? What it one remained the same while the other changed dramatically. I think we should consider any change.